jdaviz/components/layer_viewer_icon.vue

```diff
@@ -1,6 +1,11 @@
 <template>
-  <span v-if="icon !== undefined" :class="prevent_invert_if_dark ? '' : 'invert-if-dark'" :style="span_style+'; color: '+color+'; '+borderStyle">
-        {{String(icon).toUpperCase()}}
+  <span v-if="icon !== undefined">
+    <v-icon v-if="String(icon).startsWith('mdi-')" size="16">
+      {{icon}}
+    </v-icon>
+    <span v-else :class="prevent_invert_if_dark ? 'layer-viewer-icon' : 'invert-if-dark layer-viewer-icon'" :style="span_style+'; color: '+color+'; '+borderStyle">
+      {{String(icon).toUpperCase()}}
+    </span>
```

Comment:
this change is needed to allow the cursor icon in the app bar (instead of just layer letters)